goparsify v0.2.1 only exists in https://proxy.golang.org

I tried to build melange with `GOPROXY=direct` and it failed with 

```
go: downloading github.com/oec/goparsify v0.2.1
pkg/cond/parser.go:20:2: reading github.com/oec/goparsify/go.mod at revision v0.2.1: unknown revision v0.2.1
```

Taking a look at https://github.com/oec/goparsify/tags we can see there is no v0.2.1 tag (at the time of filing this issue) and the latest is v0.2.0.

When I updated to `GOPROXY=https://proxy.golang.org/,direct` I was able to build melange. 

It seems that the v0.2.1 release is just a cached artifact in the golang module proxy, is there a reason not to downgrade to v0.2.0?

--- a/go.mod
+++ b/go.mod
@@ -10,9 +10,9 @@ require (
 	github.com/go-git/go-git/v5 v5.6.1
 	github.com/google/go-cmp v0.5.9
 	github.com/google/go-containerregistry v0.14.0
+	github.com/ijt/goparsify v0.0.0-20221203142333-3a5276334b8d
 	github.com/joho/godotenv v1.5.1
 	github.com/korovkin/limiter v0.0.0-20221015170604-22eb1ceceddc
-	github.com/oec/goparsify v0.2.1
 	github.com/openvex/go-vex v0.2.0
 	github.com/package-url/packageurl-go v0.1.1-0.20220203205134-d70459300c8a
 	github.com/pkg/errors v0.9.1

--- a/pkg/cond/parser.go
+++ b/pkg/cond/parser.go
@@ -17,7 +17,7 @@ package cond
 import (
 	"fmt"
 
-	"github.com/oec/goparsify"
+	"github.com/ijt/goparsify"
 )
 
 func combineOp(n *goparsify.Result) {
@@ -106,7 +106,7 @@ func Evaluate(inputExpr string, lookupFns ...VariableLookupFunction) (bool, erro
 
 	groupChain := goparsify.Some(goparsify.Any(combinedGroup, groupOrExpr), chain).Map(collapseOp)
 
-	result, err := goparsify.Run(groupChain, inputExpr, goparsify.UnicodeWhitespace)
+	result, _, err := goparsify.Run(groupChain, inputExpr, goparsify.UnicodeWhitespace)
 	if err != nil {
 		return false, err
 	}

